tests/EvmAcceptanceTests/test/rpc/eth_estimateGas.js

```diff
@@ -0,0 +1,24 @@
+const helper = require('../../helper/GeneralHelper');
+assert = require('chai').assert;
+
+const METHOD = 'eth_estimateGas';
+
+describe("Calling " + METHOD, function () {
+
+  it("should return the estimated gas as calculated over the transaction provided", async function () {
+    const estimatedGas = 2000000000
```

Comment:
Estimated gas of 2000000000 seems weird,  looks more like gasPrice.   Typical values of estimated gas is tens of thousands for hundreds of thusand for complicated contracts.

Reply:
I mean, check if the API actually implements something sensible.

Also, this is not a full implementation of the API yet, it is more of a stub for now.

Reply:
Ganache come with defaults and that's what the test uses. The implementation is zilliqa is under construction, see ticket: https://zilliqa-jira.atlassian.net/browse/ZIL-4726